comp-sqs/src/main/java/org/jumpmind/metl/core/runtime/component/SQSReader.java

```diff
@@ -48,14 +48,15 @@
     public final static String SQS_READER_MAX_MESSAGES_TO_READ = "sqs.reader.max.messages.to.read";
     public final static String SQS_READER_DELETE_WHEN = "sqs.reader.delete.when";
     public final static String SQS_READER_READ_UNTIL_QUEUE_EMPTY = "sqs.reader.read.until.queue.empty";
```

Comment:
If we're removing the `Read Until Queue Empty` functionality, we should remove this static string as well.